R/BFF_Demux.R

```diff
@@ -8,6 +8,24 @@ utils::globalVariables(
   add = TRUE
 )
 
+#' @title Estimate Multiplet Rate
+#'
+#' @description Estimates multiplet rate by using barcode matrix dimensions and formulae from the Saijita Lab's calculator (https://satijalab.org/costpercell/)
+#' @param barcodeMatrix a barcodes-by-cells matrix of hashing counts
+#' @param num10xRuns The number of lanes the cells are loaded into (expected to be 1)
+
+.EstimateMultipletRate<- function(barcodeMatrix, num10xRuns = 1){
+  numCellsRecovered <- ncol(barcodeMatrix)
+  numCellsLoaded <- 1.75 * numCellsRecovered
```

Comment:
general code comment: I'd make this an argument with a default value of 1.75

Reply:
Good point. I added a "chemistry" variable, which guesstimates the inverse cell recovery rate based on the tables in the chemistry user guides (specifically 10k recovered cells). I'm open to ideas on whether or not this is an improvement over having a default `inverse_recovery_rate` argument.